modules/custom/lex_custom/lex_custom.module

```diff
@@ -26,4 +26,25 @@ function lex_custom_system_breadcrumb_alter(Breadcrumb &$breadcrumb, RouteMatchI
       $breadcrumb->addLink(\Drupal\Core\Link::createFromRoute($title, '<none>'));
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_update_N().
+ */
+function lex_custom_update_9401() {
```

Comment:
Give credit where you got this. I'm curious to read.

Reply:
Yea I had it in one of the links I tagged you and Matt on. I can put it here too.